src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_pending_resumption_handler.h

```diff
@@ -20,6 +19,8 @@ namespace rc_rpc_plugin {
 class RCPendingResumptionHandler
     : public resumption::ExtensionPendingResumptionHandler {
  public:
+  typedef std::map<int32_t, smart_objects::SmartObject> PendingRequestList;
```

Comment:
Move typedef closer to variable with it

Reply:
@LuxoftAKutsan 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/119/commits/1c3cb71ed51e588856fa14d5868096600d87bf0a